.github/workflows/labelsync.yml

```
@@ -1,16 +1,16 @@
-  workflow_dispatch:
-
-permissions:
-  contents: read
-  issues: write
-
-jobs:
-  label_sync:
-    uses: wolfstar-project/.github/.github/workflows/reusable-labelsync.yml@fb3b0e52ddb109fec8a64f26a9295c9e72528dc8
-    with:
-      merge-labels: true
+# name: Automatic Label Sync
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)